pkgs/tools/X11/xdg-utils/default.nix

```diff
@@ -230,6 +249,12 @@ stdenv.mkDerivation rec {
     ./allow-forcing-portal-use.patch
   ];
 
+  # enable xdg-terminal build
+  postPatch = ''
+    substituteInPlace scripts/Makefile.in \
+      --replace-fail "    xdg-settings" "    xdg-settings xdg-terminal"
```

Comment:
You want to replace https://gitlab.freedesktop.org/xdg/xdg-utils/-/blob/master/scripts/Makefile.in#L24 , right? We should probably use a patch here, otherwise we might match whatever.

Reply:
> You want to replace https://gitlab.freedesktop.org/xdg/xdg-utils/-/blob/master/scripts/Makefile.in#L24 , right? We should probably use a patch here, otherwise we might match whatever.

Thanks, have replaced with a patch file instead.